syft/execution/plan.py

```diff
@@ -382,8 +384,15 @@ def __call__(self, *args, **kwargs):
                 if _self is None:
                     response = eval(cmd)(*args, **kwargs)  # nosec
                 else:
+                    if isinstance(_self.child, crypten.mpc.MPCTensor):
```

Comment:
I guess this is temporary?
Usually the replace by the .child should be automatic